src/test/java/uk/gov/hmcts/reform/cwrdapi/service/JsrValidatorInitializerTest.java

```diff
@@ -46,4 +46,34 @@ public void testGetInvalidJsrRecords() {
         assertEquals(1, records.size());
         verify(jsrValidatorInitializer).getInvalidJsrRecords(caseWorkerProfiles);
     }
+
+    @Test
+    public void testGetNoInvalidJsrRecords_whenEmailWithMixedCases() {
```

Comment:
I think there is one major code smell which needs to be addressed - https://sonarcloud.io/project/issues?id=uk.gov.hmcts.reform%3Areform%3A%3Ard-caseworker-ref-api&open=AXhGIXbbUkYqbLveU2qp&pullRequest=234&resolved=false&severities=MAJOR&types=CODE_SMELL

Reply:
Hi Ishrar, marking this as resolved.